unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/SubSections/WorldsSubSection/WorldsSubSectionMenu/WorldsSubSectionComponentController.cs

```diff
@@ -239,8 +243,10 @@ private void OnChannelToJoinChanged(string currentChannelId, string previousChan
         OnCloseExploreV2?.Invoke();
     }
 
-    public static void JumpInToWorld(PlaceInfo worldFromAPI)
-    {
-        Environment.i.world.teleportController.JumpInWorld(worldFromAPI.world_name);
-    }
+    private void OpenWorldsDaoProposal() =>
+        WebInterface.OpenURL(DAO_PROPOSAL_LINK);
```

Comment:
You should inject the `IBrowserBridge` interface in this controller and pass `new WebInterfaceBrowserBridge()` in its initialization. We already have the ` WebInterface.OpenURL(...)` method in `IBrowserBridge` 👍